.env.example

```
@@ -83,9 +83,14 @@ NEO4J_URI=bolt://localhost:7687
 NEO4J_USERNAME=
 NEO4J_PASSWORD=
 NEO4J_DATABASE=neo4j
-NEO4J_AUTH=neo4j/bioetl_secure_password
+NEO4J_AUTH=neo4j/bioetl-neo4j-password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEO4J_AUTH key should go before the NEO4J_DATABASE key

(UnorderedKey)